deploy/cloud/modules/provision-apps/main.tf

```diff
@@ -0,0 +1,5 @@
+provider "helm" {
```

Comment:
I think the best practice for modules is to accept providers as arguments like you did with `kuberenetes`

Reply:
fixed